main.go

```diff
@@ -154,12 +155,18 @@ func githubActionOptions() (*options, error) {
 		return nil, err
 	}
 
+	filename := os.Getenv("INPUT_FILENAME")
+	if filename == "" {
+		return nil, fmt.Errorf("env var INPUT_FILENAME not set")
```

Comment:
See the comment above — in case it wasn't clear, this code is only executed when we've already determined that we are running in a GitHub action. I think it would be nicer if this binary didn't have any knowledge of GitHub actions, but that's not the way it is right now, and IMO this doesn't make that situation any worse.

Reply:
Actually, I had wrong assumptions of how running docker containers in actions worked, and there is nothing unusual here. 

Still, a default value would be useful so we could skip entirely the `input` section unless we're explicitly running it against the OWNERS. I find this behaviour better because that's 1. how it worked before so we'd compatible with previous configs and 2. the project name being codenotifer, it kind of makes sense that it's the default.

Reply:
The default value is declared [where we define the `filename` parameter in action.yml](https://github.com/sourcegraph/codenotify/pull/15/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R7). So in its current form it's compatible with previous configs.

Strictly speaking, this check is not necessary here since `INPUT_FILENAME` is always defined. It's really just a failsafe in case the parameter is renamed, removed, etc. in action.yml.

(If you'd still prefer I change this somehow, can you do it via a GitHub suggestion? I find that to be a bit easier since it's then 100% what's being suggested.)

Reply:
Han, that's the case because the other actions are using it like this: 

```
      - uses: sourcegraph/codenotify@v0.4
        env:
          GITHUB_TOKEN: ${{ secrets.CODENOTIFY_GITHUB_TOKEN }}
``` 

Sorry for losing your time, I didn't know how all of this was working. We're totally good to go then!